UKHO.ADM/UKHO.ADM.psd1

```diff
@@ -12,7 +12,7 @@
     RootModule = 'UKHO.ADM.psm1'
     
     # Version number of this module.
-    ModuleVersion = '5.0.0'
+    ModuleVersion = '6.0.0'
```

Comment:
I don't know if this is a major increment. Is there any breaking changes around the API?

Reply:
we are referencing our first required module but the call to the start is the same, so maybe 5.1.0